sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/OtherTests.java

```diff
@@ -376,6 +376,21 @@ public void testProjectFiles() throws IOException, InterruptedException {
         }
     }
 
+    @Test
+    public void duplicateColumnTest() {
```

Comment:
Nice. It looks like the reporting part for multiple errors work. 

Could you also add tests where there are multiple tables/views with errors to see if it works there too? Should probably be a different PR.

Reply:
I believe that compilation will stop after the first table with duplicate columns.
Or are you asking about multiple tables with the same name?